src/ontology/mondo-ingest.Makefile

```diff
@@ -163,6 +163,22 @@ $(COMPONENTSDIR)/icd10who.owl: $(TMPDIR)/icd10who_relevant_signature.txt | compo
 		remove -T config/properties.txt --select complement --select properties --trim true \
 		annotate --ontology-iri $(URIBASE)/mondo/sources/icd10who.owl --version-iri $(URIBASE)/mondo/sources/$(TODAY)/icd10who.owl -o $@; fi
 
+# TODO: do
+# TODO: remove any unnecessary queries / lines here
+#  - fix_hgnc_mappings
+#  - maybe make a comment about other ones i thought to delete
+$(COMPONENTSDIR)/icd11.owl: $(TMPDIR)/icd11_relevant_signature.txt | component-download-icd11.owl
```

Comment:
## Comparison to other components goals
### Removals
1. **`rename --mappings config/property-map-2.sssom.tsv `**: None of the annotation props in this file appear in ICD11. Honestly I think this may be the case for some other sources as well, even though this step appears in those goals (no harm because of `--allow-missing-entities true`)

### Kept
- **`rename --mappings config/property-map-1.sssom.tsv `**: 2/3 (`prefLabel`, `altLabel` of the annotation props appear in ICD11. However, even after the removal, the `<owl:AnnotationProperty>` declaration still remains. IDK if this is an oversight on `robot`'s part

Reply:
Reviewed with Trish today.